docusaurus.config.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
   onDuplicateRoutes: "warn",
   customFields: {
     image: "https://www.okteto.com/docs/okteto-meta-image.png",
+    kubernetesMinVersion: process.env.KUBERNETES_MIN_VERSION ?? "1.25",
```

Comment:
@rlamana how will this work across future versions? we only want this min/max value to apply to version 1.18, but for version 1.19 we might want this to be 1.26 and 1.29 (for example)

Reply:
I was just trying to figure out that, not sure how to do it just yet. I will keep investigating before merging this

Reply:
I changed the approach, it is now versioned. Read the updated comment and let me know what you think: https://github.com/okteto/docs/pull/640#issue-2137013176